examples/kubernetes/README.md

```diff
@@ -0,0 +1,75 @@
+# DigitalOcean Kubernetes Service (DOKS)
+
+This example shows how to use the Terraform Kubernetes Provider and Terraform
+Helm Provider to configure a DOKS cluster. The example config builds the DOKS
+cluster and applies the Kubernetes configurations in a single operation. This
+guide will also show you how to make changes to the underlying DOKS cluster in
+such a way that Kuberntes/Helm resources are recreated after the underlying
+cluster is replaced.
+
+You will need to set the following environment variables:
+
+ - `DIGITALOCEAN_ACCESS_TOKEN`
+
+To install the DOKS cluster using default values, run terraform init and apply
+from the directory containing this README.
+
+```
+terraform init
+terraform apply
+```
+
+Optionally, the Kubernetes version, the number of worker nodes, and the instance
+type of the worker nodes can also be specified:
+
+```
+terraform apply -var=cluster_version=1.18 -var=worker_size=s-4vcpu-8gb -var=worker_count=5
+```
+
+
+## Versions
+
+Valid versions for the DOKS cluster can be found by using the doctl, the DigitalOcean CLI.
+
+```
+ doctl kubernetes options versions
+```
+
+## Kubeconfig for manual CLI access
+
+Optionally, this example can generate a kubeconfig file in the current working
+directory. However, the token in this config will expire. The token can be
+refreshed by running `terraform apply` again and setting `write_kubeconfig` to
+`true`.
+
+```
+terraform apply -var="write_kubeconfig=true"
+export KUBECONFIG=$(terraform output -raw kubeconfig_path
+kubectl get pods -n test
+```
```

Comment:
I use a combination of `terraform refresh` and `terraform output kubeconfig > .../kubeconfig`. That seems less intimidating (i.e. potentially destructive) that `terraform apply`.

Reply:
Since this creates a local_file resource, apply is needed to replace the file if new credentials are needed.